webrtc.html

```diff
@@ -1223,16 +1223,25 @@ <h4>Set the RTCSessionDescription</h4>
             <p data-tests="">In parallel, start the process to apply <var>description</var>
             as described in <span data-jsep=
             "processing-a-local-desc processing-a-remote-desc">[[!JSEP]]</span>,
-            with these additional restrictions: Use <var>jsepSetOfTransceivers</var>
-            as the source of truth with regard to what "RtpTransceivers" exist, and
-            their <a>[[\JsepMid]]</a> internal slot as their "mid property".
-            If applying <var>description</var>
-            leads to modifying a transceiver <var>transceiver</var>, and
-            <var>transceiver</var>.<a>[[\Sender]]</a>.<a>[[\SendEncodings]]</a>
-            is non-empty, and not equal to the encodings that would result from
-            processing <var>description</var>, the process of applying <var>description</var>
-            fails. This specification does not allow remotely initiated RID renegotiation.</p>
+            with these additional restrictions:<p>
             <ol>
+              <li>
+                <p>Use <var>jsepSetOfTransceivers</var> as the source of truth
+                with regard to what "RtpTransceivers" exist, and their
+                <a>[[\JsepMid]]</a> internal slot as their "mid property".</p>
+              </li>
+              <li>
+                <p>If <var>remote</var> is <code>true</code>, validate
+                back-to-back offers as if answers were applied in between.</p>
```

Comment:
I'm not sure what "as if answers were applied in between" means?
Notably, we are not saying to apply mock answers in between because that may lead to transmission, right? This only refers to validation?

Can this be clarified? Is it supposed to say "validate as if we are in the stable state" or...?

Reply:
Prose was based on https://github.com/w3c/webrtc-pc/issues/2512#issuecomment-614694127. How about adding _", by running the check for subsequent offers in stable state."_?